### ISSUE TITLE
Grenade settings get reset while hog dies 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#450]
```
When playing the highlander mode (or other unlimited attack games), timers and 
"bounce"-settings for greandes get reset if a hog dies.

Example: A hog1's health is brought down to zero. You pick a grenade for the 
next attack, while hog1 did not die yet. You choose 1 second and low bounce for 
the grenade. hog1 dies now, and grenade settings are back to 3 seconds and 
normal bounce.

```

Original issue reported on code.google.com by `tobiasla...@gmail.com` on 29 Oct 2012 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Appendix:

In Beta 0.9.18, also the weapen menu closes when the hog dies.
```

Original comment by `tobiasla...@gmail.com` on 29 Oct 2012 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2af25494f3bb.
```

Original comment by `unC0Rr` on 16 Aug 2013 at 10:16
* Changed state: **Fixed**
